src/element/collision.ts

```diff
@@ -332,3 +333,10 @@ const hitTestRoughShape = (
     return false;
   });
 };
+
+function isElementSelected(
```

Comment:
could we use arrow function instead?

Reply:
Sure! I'll do it in a moment :)

Reply:
> It's not a huge change, so we could even merge it and refactor it in a another PR..

Change is done :) Let me know if something else is needed!